tensorflow_datasets/image/coil100.py

```diff
@@ -0,0 +1,87 @@
+"""Dataset class for COIL-100 dataset"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+import os
+import fnmatch
+
+_URL = "http://www.cs.columbia.edu/CAVE/databases/SLAM_coil-20_coil-100/coil-100/coil-100.zip"
+
+_DESCRIPTION = ("""The dataset contains 7200 color images of 100 objects 
+(72 images per object). The objects have a wide variety of complex geometric and reflectance characteristics.
+The objects were placed on a motorized turntable against a black background.
+The turntable was rotated through 360 degrees to vary object pose with respect to a fxed color camera.
+Images of the objects were taken at pose intervals of	5 degrees.This corresponds to
+72 poses per object""")
+
+_LABELS = [str(x) for x in range(0,360,5)]
+
+_IMAGE_SHAPE = (128,128,3)
+
+_CITATION = """\
+
+@article{nene1996columbia,
+  title={Columbia object image library (coil-20)},
+  author={Nene, Sameer A and Nayar, Shree K and Murase, Hiroshi and others},
+  year={1996},
+  publisher={Technical report CUCS-005-96}
+}
+
+"""
+
+class Coil100(tfds.core.GeneratorBasedBuilder):
+  """COIL-100 Image Dataset Class"""
+
+  VERSION = tfds.core.Version('1.0.0')
+
+  def _info(self):
+        """Define Dataset Info"""
+
+        return tfds.core.DatasetInfo(
+            builder=self,
+    
```

Comment:
Why the extra line here ? It seems you have this in all your PR ?

Reply:
Yes, I missed those standard specifications, sorry for that. I will fix them ASAP.